plugins/callback/opentelemetry.py

```diff
@@ -0,0 +1,444 @@
+# (C) 2021, Victor Martinez <VictorMartinezRubio@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    author: Victor Martinez (@v1v)  <VictorMartinezRubio@gmail.com>
+    name: opentelemetry
+    type: notification
+    short_description: Create distributed traces with OpenTelemetry
+    version_added: 3.7.0
+    description:
+      - This callback create distributed traces for each Ansible task with OpenTelemetry.
+      - You can configure the OTEL exporter with environment variables. See U(https://opentelemetry-python.readthedocs.io/en/latest/exporter/otlp/otlp.html).
+    options:
+      hide_task_arguments:
+        default: false
+        type: bool
+        description:
+          - Hide the arguments for a task.
+        env:
+          - name: OPENTELEMETRY_HIDE_TASK_ARGUMENTS
```

Comment:
I don't see any of these environment variables in the documentation in https://opentelemetry-python.readthedocs.io/en/latest/exporter/otlp/otlp.html, so I would assume they are ansible specific? In that case, they should be prefixed with `ANSIBLE_`. (Also two are prefixed with `OPENTELEMETRY_` right now, and one with `OTEL_`. Is that intentional?)

Reply:
OPENTELEMETRY_HIDE_TASK_ARGUMENTS and OPENTELEMETRY_CONSOLE_OUTPUT are ansible specific, so I'll change them with the prefix `ANSIBLE_`. 

While `OTEL_SERVICE_NAME` is opentelemetry specific, see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#general-sdk-configuration

Since it's default value is `ansible` that's the reason I added explicitly there.

Reply:
See https://github.com/ansible-collections/community.general/pull/3091/commits/5d13e07303573c83c69ce677f43a572339711060 in conjunction with https://github.com/ansible-collections/community.general/pull/3091/commits/2023e15641279f53cb662aa68d486823c9a70e31 to clarify the above-mentioned environment variable.